lms/djangoapps/ora_staff_grader/views.py

```diff
@@ -147,6 +150,67 @@ def get(self, request, ora_location, *args, **kwargs):
             return UnknownErrorResponse()
 
 
+class AssessmentFeedbackView(StaffGraderBaseView):
+    """
+    GET data about Assessments by submission_uuid and ora_location
+
+    Response: {
+        assessments (list of dict): [
+            {
+                "assessment_id: (string) assessment id
+                "scorer_name: (string) scorer name
+                "scorer_username: (string) scorer username
+                "scorer_email: (string) scorer email
+                "assessment_date: (string) assessment date
+                "assessment_scores (list of dict) [
+                    {
+                        "criterion_name: (string) criterion name
+                        "score_earned: (int) score earned
+                        "score_type: (string) score type
+                    }
+                ]
+                "problem_step (string) problem step (Self, Peer, or Staff)
+                "feedback: (string) feedback
+            }
+        ]
+    }
+
+    Errors:
+    - MissingParamResponse (HTTP 400) for missing params
+    - BadOraLocationResponse (HTTP 400) for bad ORA location
+    - XBlockInternalError (HTTP 500) for an issue with ORA
+    - UnknownError (HTTP 500) for other errors
+    """
+
+    @require_params([PARAM_ORA_LOCATION, PARAM_SUBMISSION_ID, PARAM_ASSESSMENT_FILTER])
+    def get(self, request, ora_location, submission_uuid, assessment_filter, *args, **kwargs):
+
+        try:
+            assessments_data = {
+                "assessments": get_assessments(
+                    request, ora_location, submission_uuid, assessment_filter
+                )
+            }
+
+            response_data = AssessmentFeedbackSerializer(assessments_data).data
+            return Response(response_data)
+
+        # Catch bad ORA location
```

Comment:
Remember not to use descriptive in-line comments

Reply:
In-line comments deleted